fragment.go

```diff
@@ -2007,7 +2013,7 @@ func (f *fragment) importRoaring(ctx context.Context, data []byte, clear bool) e
 	f.mu.Lock()
 	defer f.mu.Unlock()
 	span.Finish()
-	bm := roaring.NewBTreeBitmap()
+	bm := roaring.NewSliceBitmap()
```

Comment:
I think this will cause poor performance in the case where we Union `f.storage` into `bm` instead of the other way around (see line 2072)

Reply:
That is possible, but that should only happen when the new bitmap is the larger of the two, in which case the stuff we're unioning in is comparatively small. So I suspect that will be a rareish case, and/or a small case.

Reply:
it turns out that this analysis became completely wrong when i switched to the "just use the bitmap we just wrote to storage as the bitmap" thing, and produced some 1000% or larger performance regressions in this patch set, so.